src/form-builder/FullNameField.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { FieldProps } from './types';
+import SelectField from './SelectField';
+import TextField from './TextField';
+
+const FullNameField = (props: FieldProps<string>): JSX.Element => {
+  const suffixes = ['', 'Jr.', 'Sr.', 'II', 'III', 'IV']
```

Comment:
Can this be moved outside of the component body? Ideally, we should import this from a constants file, maybe `utils/contants.ts`?

Reply:
Yes, that sounds right place for it.